src/usbguard-icon.svg

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg:svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="61.17897pt"
+   height="61.671368pt"
+   viewBox="0 0 61.17897 61.671368"
+   version="1.1"
+   id="svg35"
+   sodipodi:docname="usbguard-icon.svg"
+   inkscape:version="0.92.4 5da689c313, 2019-01-14">
+  <svg:metadata
+     id="metadata41">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title />
+      </cc:Work>
+    </rdf:RDF>
+  </svg:metadata>
+  <svg:defs
+     id="defs39" />
+  <sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="1375"
+     inkscape:window-height="711"
+     id="namedview37"
+     showgrid="false"
+     fit-margin-top="0"
+     fit-margin-left="0"
+     fit-margin-right="0"
+     fit-margin-bottom="0"
+     inkscape:zoom="3.4471456"
+     inkscape:cx="-11.052196"
+     inkscape:cy="34.256267"
+     inkscape:window-x="440"
+     inkscape:window-y="720"
+     inkscape:window-maximized="0"
+     inkscape:current-layer="svg35" />
+  <script>(function () {
+  var rand = {
+    &quot;noise&quot;: function () {
+      var SIGN = Math.random() &lt; Math.random() ? -1 : 1;
+      return Math.floor(Math.random() + SIGN * Math.random());
+    },
+    &quot;sign&quot;: function () {
+      const tmp = [-1, -1, -1, -1, -1, -1, +1, -1, -1, -1];
+      const index = Math.floor(Math.random() * tmp.length);
+      return tmp[index];
+    }
+  };
+  //
+  Object.defineProperty(HTMLElement.prototype, &quot;offsetHeight&quot;, {
```

Comment:
wtf is this?
Is this coming from upstream?

Reply:
You mean the SVG?
I just downloaded the logo from https://usbguard.github.io/
And adjusted it so it would be only the logo and not the text and saved it.

The SVG provides much better quality regarding scaling for desktop/launcher/tray icon.